modules/gitops-enabling-dex.adoc

```diff
@@ -0,0 +1,48 @@
+// Module is included in the following assemblies:
+//
+// * 
+
+[id="enabling-dex_{context}"]
+= Enabling Dex
+
+Argo CD embeds and bundles Dex as part of its installation. Dex is an identity service that uses OpenID Connect to drive authentication for other apps.
+
+.Procedure
+
+. Enable Dex by updating the Subscription resource for the OpenShift GitOps Operator.
++
+[source,yaml]
+----
+spec:
+  config:
+    env:
+    - name: DISABLE_DEX
+      Value: "false"
+----
++
+This will get the `argocd-cluster-dex-server` instance running. 
```

Comment:
Just a suggestion:
This update causes the `argocd-cluster-dex-server` instance to run.